src/main/java/com/github/javafaker/service/FakeValues.java

```diff
@@ -70,12 +70,12 @@ private Map loadValues() {
             return null;
         }
 
-        final Map valuesMap = new Yaml().loadAs(stream, Map.class);
-        Map localeBased = (Map) valuesMap.get(locale.getLanguage());
+        final Map<?,?> valuesMap = new Yaml().loadAs(stream, Map.class);
```

Comment:
could you stick to one logical change per PR please? I'm happy to add the babylon5 but if you wanted to refactor the Map, please make it a different PR